### PR TITLE
Docu Update

### PR DESCRIPTION
**Pull request by newLabAspect**
- minor updates User Guide
- remove outdated description Coffee Machine
- replace outdated description Failure Time Prediction
- minor corrections One Tank

### CODE REVIEW

File: docs/examples/coffee_machine.md [hunk 1]
```diff
@@ -1,57 +1,31 @@
...
-learners:
-  - name: Automaton Learner
-    class_path: agenc.learners.grpc.GrpcLearner
-    arguments:
-      port: 8080
-      ip: "localhost"
-```
+## Local Execution
 
+It is also possible to directly run the Java server locally.
```
Comment (schmidma) [inline docs/examples/coffee_machine.md:25]: Please put one line per sentence. This makes line based version diff much easier to read and maintain

File: docs/examples/failure_time_prediction.md [hunk 2]
```diff
@@ -30,20 +30,30 @@ The `preprocessing.ipynb` notebook is used to reproduce the steps and generate a
...
+
+### From source
+
+Follow the [installation guide](../getting_started/installation.md) to install flowcean and it's dependencies from source.
+Afterwards you can navigate to the `examples` folder and run the examples.
+
+```sh
+cd examples/failure_time_prediction
+python run.py
+```
```
Comment (schmidma) [inline docs/examples/failure_time_prediction.md:59]: Is this a proper alternative? I'd just use `hatch shell`. Then hatch again takes care of setting things up, including installing dependencies etc.

File: docs/user_guide/environment.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ classDiagram
   }
   class IncrementalEnvironment{
     Loads data in an iterative way. 
-    + get_next_data() -> DataFrame
+    + collect() -> DataFrame
```
Comment (schmidma) [inline docs/user_guide/environment.md:31]:
```suggestion
    + collect(n: int) -> DataFrame
```

**Review by schmidma — CHANGES_REQUESTED** [review]

File: docs/examples/failure_time_prediction.md (hunk 2 above)
Comment (newLabAspect) [reply docs/examples/failure_time_prediction.md:59]: As we explain how to install flowcean yourself, I like having the explanation how to run the example in your current python environment. We also mention that using `hatch run` is the easier option.

But I am also fine with removing this, if you consider it confusing or unnecessary.

File: docs/examples/coffee_machine.md (hunk 1 above)
Comment (newLabAspect) [reply docs/examples/coffee_machine.md:28]: Done. Also for all (if I didn't miss something) other docu files.

**Pull request merged**